include/cpp-statsd-client/StatsdClient.hpp

```diff
@@ -59,28 +60,64 @@ class StatsdClient {
     const std::string& errorMessage() const noexcept;
 
     //! Increments the key, at a given frequency rate
-    void increment(const std::string& key, float frequency = 1.0f) const noexcept;
+    void increment(const std::string& key,
```

Comment:
since i added tags, all of the method signatures changed. its hard to check over but thats the only diff to the method signatures